pkgs/applications/science/biology/dcm2niix/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libyamlcpp
+}:
+
+let
```

Comment:
Since you are using a recursive attribute set you might as well skip the `let` block entirely and put these attributes on the derivation attribute set.